src/documents/ImageReordererAPI.md

```diff
@@ -237,7 +236,6 @@ selectors: {
     selectables: ".flc-reorderer-movable",
     dropTargets: ".flc-reorderer-movable",
     grabHandle:  "",
```

Comment:
There's not a trailing comma after `grabHandle: ""`

Reply:
oops, will fix it soon :)